clients/includes/install-message-factory.inc

```diff
@@ -14,6 +14,7 @@ accessed through :ref:`factories <message-factory>`. Message factories for
 
     $ composer require php-http/message
 
-.. _Diactoros: https://github.com/zendframework/zend-diactoros
 .. _Guzzle PSR-7: https://github.com/guzzle/psr7
+.. _Laminas Diactoros: https://github.com/laminas/laminas-diactoros
 .. _Slim Framework: https://github.com/slimphp/Slim
+.. _Zend Diactoros: https://github.com/zendframework/zend-diactoros
```

Comment:
the failure is now because the references in the text say only "Diactoros" but you renamed it to "Zend Diactoros" (which is the correct thing to do, to avoid confusion)

```
clients/includes/install-message-factory.inc:8: ERROR: Unknown target name: "diactoros".
```

can you please rename on line 8 of this file as well?

Reply:
seems to be line 10 actually. lets say 
```
`Laminas Diactoros`_ (and it's abandoned predecessor `Zend Diactoros`_)
```

Reply:
Oh I see, so when writing something like this:
```
`Laminas`_ 
```

It is a reference to the links provided at the bottom of the file:
```
.. _Laminas: https://example.org
```

Right?